adapter/src/main/java/org/jvalue/ods/adapterservice/datasource/event/DatasourceCreationEvent.java

```diff
@@ -0,0 +1,41 @@
+package org.jvalue.ods.adapterservice.datasource.event;
+
+import org.jvalue.ods.adapterservice.datasource.model.Datasource;
+import org.jvalue.ods.adapterservice.datasource.model.DatasourceTrigger;
+
+import java.util.Objects;
+
+public class DatasourceCreationEvent extends DatasourceConfigEvent {
+
+    private final DatasourceTrigger trigger;
```

Comment:
Probably it is better to send the whole config.
Clients will only extract the parts they need then (e.g. the scheduler extracts the trigger config)

Reply:
Good point, will change it accordingly